index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+// const fs = require('fs');
+// const expect = require('chai').expect;
+const readfiles = require('./lib/read-files.js');
+const main = module.exports = function (argv){
+    if(argv.length !=5) return 'Please enter at least 3 arguments';
+        return readfiles(argv.slice(2,5), function(err, data){
+            if (err) {
+                return console.error(err);
+            }
+            return console.log(data);
+    })
```

Comment:
My linter was giving me missing semi-colon and space issues with your code. I know you're using a different text editor, so I'm not sure if your eslintrc is flagging things properly for you??

In future labs, Duncan might utilize a tool called "Travis" which basically reads through your pull request to make sure you have passing tests and you have no linter errors. 

If these apply, your PR will fail, and I'm not allowed to even look at it for review and will just give an automatic 0 (though, of course, I can still give full points for resubmissions assuming the first submission was on time). 

So just an FYI to make sure you're getting correct linter errors with your text editor before we run into any future problems. :)